MericCelik/assignment1/LinkedList.java

```diff
@@ -0,0 +1,61 @@
+
+public class LinkedList<T>{
```

Comment:
Very nice usage of Generics. :) Good job!

Reply:
I'm glad to hear that! :relaxed: